task/git-clone/0.1/git-clone.yaml

```
@@ -205,6 +211,12 @@ spec:
       fi
       printf "%s" "${RESULT_SHA}" > "$(results.commit.path)"
       printf "%s" "${PARAM_URL}" > "$(results.url.path)"
+
+      if [ "${PARAM_FETCH_TAGS}" = "true" ] ; then
+        echo "Fetching tags"
+        git fetch --tags
```

Suggestion. The presence of `PARAM_DELETE_EXISTING` which can be `false` tells that git repo dir can be reused. Not sure in which contexts, but if it gets reused and tags get changed in the meantime, just `git fetch --tags` does not refresh the ones present in local copy. The ones that got moved in remote won't be adjusted locally without `--force` flag. Adding `--force` will make sure that local tags will be refreshed from remote.

```suggestion
        git fetch --tags --force
```

---

That's a good point. I would leave it to the maintainers to confirm your assumption about `PARAM_DELETE_EXISTING`. If in fact this task is meant to run against a checked out repository, `--force` would be appropriate.

---

@mmorhun could you comment on this point?